src/dotnet/APIView/APIViewWeb/Client/src/review.ts

```diff
@@ -270,6 +270,7 @@ $(() => {
             }
           }
         }
+        navItemRow.removeClass("nav-list-collapsed");
```

Comment:
it looks like we also have to do this no matter the if statement is executed or not. could we remove it from here and the `else` and applied it one time?

Reply:
yes. I think so.  I will make that change